client/src/test/App.test.tsx

```
@@ -0,0 +1,50 @@
+
+// Create a query client for testing
+const queryClient = new QueryClient({
+  defaultOptions: {
+    queries: {
+      retry: false,
+    },
+  },
+});
+
+// Helper to render App with necessary wrappers
+function renderWithProviders(initialRoute: string = '/') {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)